udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -21,17 +21,38 @@
 import java.util.Set;
 
 /**
- * Returns a collection of TimeRange objects to indicate available meeting times
- * for indicated people and duration given through a request object.
+ * Contains methods for querying availability between attendees.
  */
 public final class FindMeetingQuery {
   private static final int OVERLAP_WITH_SAME_START = 1;
   private static final int OVERLAP_WITH_SAME_END = 2;
   private static final int OVERLAPS_END = 3;
   private static final int OVERLAPS_START = 4;
   private static final int OVERLAP_CONTAINS = 5;
+  private static final int NO_OVERLAP = 0;
```

Comment:
How about introducing an enum instead? This would improve the semantics of these constants (it would make it clearer that these belong to a related set of mutually exclusive values).

Reply:
Done, was thinking of a way to avoid the integers, great change.